### PR TITLE
Add command summary to help window

### LINKED ISSUE #71: Add command summary to the help window
Aim to guide users on the command format without navigating to User Guide website since it is an offline app.

### PR DESCRIPTION
**Pull request by Bonzzz3**
Ready for review
Fixes #71 

### **Description**
Update help window

### **Key Changes**
- Add command summary to help window
- Change format and example column to text area to allow selection and copy for users
- Update user guide with enhanced help feature
- Add tests for command summary

### **Future Plan**
Update command summary in help window and in user guide with new features

### CODE REVIEW

**Review by ritvi12 — APPROVED** [review]
Other than these changes if needed, everything else seems okay!

**Review by myrilla12 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -99,4 +116,57 @@ private void copyUrl() {
...
+                        + "a/Hickory Forest b/23 s/Female ap/11/11/2024 1100"),
+                new CommandSummary("Clear", "clear"),
+                new CommandSummary("Delete", "delete INDEX / delete NAME\ne.g., delete 3, delete Alex Yeoh"),
+                new CommandSummary("Edit", "edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/AGE] [s/SEX] "
+                        + "[ap/APPOINTMENT] [t/TAG]\ne.g., edit John Doe n/Betsy Crower t/ ap/"),
+                new CommandSummary("Find", "find KEYWORD [MORE_KEYWORDS]\ne.g., find olive 87438"),
+                new CommandSummary("Help", "help"),
+                new CommandSummary("List", "list")
+        );
+
```
Comment (LynetteLeeSiYing) [inline src/main/java/seedu/address/ui/HelpWindow.java:136]: Good to have [square brackets] to indicate optional field! Do we need a side note on that? an exaplanation of what square bracket, UPPERCASE means?

**Review by LynetteLeeSiYing — APPROVED** [review]
LGTM! Just some additional notes

File: docs/images/helpwindow.png
Comment (ritvi12) [inline docs/images/helpwindow.png:1]: Can adapt the ui to be similar to the theme of the MediContact theme colors.

**Review by ritvi12 — APPROVED** [review]
Can make changes if needed.

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (Bonzzz3) [reply src/main/java/seedu/address/ui/HelpWindow.java:136]: I just copied from the changes made in User Guide. I will update the command summary when the User Guide on website made changes. I think we should discuss the format. Then, I will add side note accordingly.

File: docs/images/helpwindow.png
Comment (Bonzzz3) [reply docs/images/helpwindow.png:1]: Alright, I will do that for our next milestone.

**Pull request merged**